pkg/certificates/triple/cert/cert.go

```diff
@@ -0,0 +1,269 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```

Comment:
I guess this got removed from the client-go library?

Reply:
Yup

Reply:
> Yup

/hold cancel